internal/proto/local/controller/api/services/v1/auth_method_service.proto

```diff
@@ -211,8 +210,12 @@ message AuthenticateRequest {
 }
 
 message AuthenticateResponse {
-  resources.authtokens.v1.AuthToken item = 1;
-  string token_type = 2 [json_name="token_type"];
+  reserved 1, 2; // Old item and token_type
+  reserved "item", "token_type";
+  // Valid keys and values depend on the type of Auth Method as well as the command.
+  google.protobuf.Struct attributes = 4 [json_name="attributes"];
```

Comment:
I wonder if you want to do a similar thing you did with PasswordCredentials for `PasswordResponseAttributes` to document what the structure of a password specific AuthenticateResponse attributes is.  This will be helpful with tests, the outgoing interceptor, and for anyone looking at the code trying to figure out what should be in the response.  It might also be helpful for future plugins to have a way to document what the expected request and response look like.

Reply:
The structure of the password specific authenticate response attributes is already defined -- it's an AuthToken.

I think we could potentially add more of these in the future but I'd prefer to defer for now.